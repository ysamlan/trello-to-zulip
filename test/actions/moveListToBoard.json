{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One"
    },
    "boardSource": {
      "id": "b2idb2idb2idb2idb2idb2id",
      "name": "Board Two"
    },
    "list": {
      "id": "lidlidlidlidlidlidlidlid",
      "name": "List Name"
    },
    "member": {
      "id": "mcidmcidmcidmcidmcidmcid",
      "name": "Member Creator Full Name"
    }
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "moveListToBoard"
}