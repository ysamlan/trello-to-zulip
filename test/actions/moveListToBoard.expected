Member Creator Full Name performed moveListToBoard on [Board One](https://trello.com/board/b1idb1idb1idb1idb1idb1id)